src/lib/queries/json.ts

```diff
@@ -14,7 +14,7 @@ export class JsonCommonQuery implements CommonQuery {
 
 	private client: Client;
 	private get provider() {
-		return this.client.providers.get('json') as JsonProvider;
+		return (this.client.providers.get('json') ?? this.client.providers.get('cache')) as JsonProvider;
```

Comment:
This breaks when the cache provider is used over the JSON provider, as the JSON provider will still be loaded.